src/tests/ilp-kit.js

```diff
@@ -104,4 +149,127 @@ describe('ILP Kit Test Suite -', function () {
+        .auth('alice', 'alice')
+        .set('Content-Type', 'application/json')
+        .send({
+          destination: 'bob@wallet1.example:443',
+          destinationAmount: 1,
+          sourceAmount: 1,
+          message: 'intraledger payment test'
+        })
+      assertStatusCode(resp, 200)
+      yield kitManager.assertBalance(kitManager.kits[0], 'alice', '999')
+      yield kitManager.assertBalance(kitManager.kits[0], 'bob', '1001')
+    })
+
+    it.skip('Make an interledger payment (same currency)', function * () {
+      const config = kitManager.kits[0]
+      const resp = yield request
+        .put(`https://${config.API_HOSTNAME}:${config.API_PUBLIC_PORT}/api/payments/aaaa70ec-08b9-11e6-b512-3e1d05defe78`)
+        .auth('alice', 'alice')
+        .set('Content-Type', 'application/json')
+        .send({
+          destination: 'bob@wallet2.example:443',
+          destinationAmount: 100,
+          message: 'interledger payment test'
+        })
+      assertStatusCode(resp, 200)
+
+      // Alice should have:
+      //    1000      USD
+      //  -  100      USD (destinationAmount for Bob)
+      //  -    1      USD (fee connie@wallet2: 1% of destinationAmount)
+      //  -    1.01   USD (fee connie@wallet1: 1% of (destinationAmount + fee connie@wallet2)
+      //  ===============
+      //     897,99   USD
+
+      // The sourceAmount returned by a quote amount is deviating more than
+      // 2 cents for a payment of 100. It is unclear whether this
+      // is due to the imprecision inherent to floating-point operations or
+      // an error in the calculation of the quote.
+      //
+      // TODO: Investigate if the deviation from the expected sourceAmount is ok.
+      // Assume for now the deviation is ok.
+      const expectedSourceAmount = 102.01
+      const epsilon = expectedSourceAmount * 0.0003
+
+      yield kitManager.assertBalance(kitManager.kits[0], 'alice', 897.99, epsilon)
+      yield kitManager.assertBalance(kitManager.kits[1], 'bob', 1100)
+      yield kitManager.assertBalance(kitManager.kits[0], 'connie', 1102.01, epsilon)
+      yield kitManager.assertBalance(kitManager.kits[1], 'connie', 900)
+    })
+
+    it.skip('Make an interledger payment (cross-currency)', function * () {
+      const config = kitManager.kits[1]
+      const resp = yield request
+        .put(`https://${config.API_HOSTNAME}:${config.API_PUBLIC_PORT}/api/payments/bbbb70ec-08b9-11e6-b512-3e1d05defe78`)
+        .auth('alice', 'alice')
+        .set('Content-Type', 'application/json')
+        .send({
+          destination: 'bob@wallet3.example:443',
+          destinationAmount: 100,
+          message: 'interledger payment test'
+        })
+      assertStatusCode(resp, 200)
+      // TODO: configure a static exchange rate and assert that the balances match
+    })
+
+    it('Reject a payment with missing destinationAmount', function * () {
```

Comment:
Agreed, the criterion should be: If the components don't talk to each other or only in a very trivial way, it should be a repo-specific test, not an E2E integration test.